src/providers/mod.rs

```diff
@@ -98,7 +98,6 @@ fn write_ssh_keys(user: User, ssh_keys: Vec<PublicKey>) -> Result<()> {
 #[cfg(not(feature = "cl-legacy"))]
 fn write_ssh_keys(user: User, ssh_keys: Vec<PublicKey>) -> Result<()> {
     use std::io::ErrorKind::NotFound;
-    use tempfile;
```

Comment:
How is it redundant? Is it imported in some other file?

Reply:
and this shows:
```shell
warning: the item `tempfile` is imported redundantly
   --> src/providers/mod.rs:101:9
    |
101 |     use tempfile;
    |         ^^^^^^^^
    | 
   ::: src/main.rs:39:1
    |
39  | extern crate tempfile;
    | ---------------------- the item `tempfile` is already imported here
    |
    = note: `#[warn(unused_imports)]` on by default
```